### PR TITLE
Webhook updates

### PR DESCRIPTION
**Pull request by olivier34000**
* [x] webhooks supports only utf8
* [x] add thread_id doc

### CODE REVIEW

File: _docs/incoming-webhooks.md [hunk 1]
```diff
@@ -65,4 +65,43 @@ just need to copy/paste it and change the token (XXX...) by yours.
       }
     }' https://webhook.talkspirit.com/v1/incoming/XXXXXXXXXXXXXXXXXXXXXXXX
 
+## Thread id field
```
Comment (jeremyb) [inline _docs/incoming-webhooks.md]: @olivier34000 another version could be:

```markdown
## Threaded message:

talkSpirit supports threads, so you can send many webhooks on the same post. The following 
webhooks are added as comments of the initial post.

To handle threaded messages you just have to define an identifier and set it in the payload in 
the field `thread_id`. You can for example set the today's date or any unique identifier you want.
```

File: _posts/2016-06-01-circleci-webhook.md [hunk 2]
```diff
@@ -20,13 +20,13 @@ notify:
     - url: https://webhook.talkspirit.com/v1/incoming/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 ```
 
-Les notifications déclenchées sont :
+Les notifications sontdéclenchées :
```
Comment (jeremyb) [inline _posts/2016-06-01-circleci-webhook.md]: `sont déclenchées`

Comment (olivier34000) [inline _posts/2016-06-01-circleci-webhook.md]: ok

File: _docs/incoming-webhooks.md (hunk 1 above)
Comment (olivier34000) [inline _docs/incoming-webhooks.md]: ok

**Pull request merged**